src/components/UserList/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import { Table,Jumbotron,Col,Clearfix,Button } from "react-bootstrap";
+import { Link } from 'react-router-dom';
+
+class TableDataRow extends Component {
+    render() {
+        return (
+            <tr>
+                <td>
+                    <Link to={"/user/" + this.props.data.id}>{this.props.data.name}</Link>
+                </td>
+                <td>{this.props.data.email}</td>
+                <td>{this.props.data.address.street + ", " + this.props.data.address.suite + 
+                    ", " + this.props.data.address.city}</td>
+                <td>{this.props.data.company.name}</td>
+                <td>
+                    <Button bsStyle="link" onClick={this.delete.bind(this)}> 
+                        <i className="glyphicon glyphicon-trash action-link" />
+                    </Button>
+             </td>
+            </tr>
+        );
+    }
+    delete(){
+        this.props.delete(this.props.data.id);
+    }
+}
+
+class UserList extends Component {
+    constructor(props) {
+        super(props);
+        this.state={
+            userList: []
+        };
+    }
+    render() {
+        let table = 
+        this.state.userList.map((user ,i)=>{
+            return ( <TableDataRow data={user} key={i}
```

Comment:
The <TableDataRow> component should be written in one single line or into multiple lines as follows for better readability.
```
<TableDataRow
  data={user}
  key={i}
  delete={this.deleteEntry.bind(this)}
/>
```